test/e2e/primaryoption.spec.ts

```diff
@@ -1,57 +1,124 @@
 /// <reference types="../../cypress/support" />
 
 describe("Primary option page", () => {
-  describe("when user is successfully logged-in", () => {
-    it("should show primary option page once", () => {
-      cy.login();
+  it("should show primary option page once", () => {
+    cy.login();
+    // WHEN I successfully login
+    cy.visit("/anmelden/erfolgreich");
+    // THEN I click continue button
+    cy.contains("a", "Verstanden & weiter").click();
+    // THEN I should see bundesident primary option page
+    cy.contains(
+      "h1",
+      "Möchten Sie sich in wenigen Minuten mit Ihrem Ausweis identifizieren?"
+    );
+    // THEN I click Identifikation mit Ausweis
+    cy.contains("a", "Identifikation mit Ausweis").click();
+    // WHEN I successfully login second time
+    cy.visit("/anmelden/erfolgreich");
+    // THEN I click continue button
+    cy.contains("a", "Verstanden & weiter").click();
+    // THEN I should not see bundesident primary page
+    cy.contains("h1", "Mit welcher Option möchten Sie sich identifizieren?");
+    cy.url().should("include", "/identifikation");
+  });
+
+  describe("user clicks on sidebar identifikation button", () => {
+    it("should see bundesident once", () => {
       // WHEN I successfully login
+      cy.login();
+      cy.viewport(1200, 1000); // element is invisible, we need to set the screen bigger
```

Comment:
this I do not really get...

Reply:
Ah, okay. Thanks We could also set it in beforeEach then, right?

Reply:
ah right ! i missed that. thanks.